error when importing cloudflare notifcation policy with filters

### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [x] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

1.1.8

### Affected resource(s)

cloudflare_notification_policy

### Terraform configuration files

```hcl
resource "cloudflare_notification_policy" "terraform_managed_resource" {
  account_id = "58c27beae56f40b5b37a61c651337d04"
  name = "Origin Error Rate Alert"
  enabled = true
  alert_type = "http_alert_origin_error"
}
```


### Debug output

```LOG
2022/04/20 13:53:15 [WARN] Log levels other than TRACE are currently unreliable, and are supported only for backward compatibility.
  Use TF_LOG=TRACE to see Terraform's internal logs.
  ----
2022/04/20 13:53:15 [INFO] Terraform version: 0.14.7  
2022/04/20 13:53:15 [INFO] Go runtime version: go1.15.6
2022/04/20 13:53:15 [INFO] CLI args: []string{"/usr/local/bin/terraform", "import", "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721", "01cc14d1b817a0abc6f8f278499b0633/4f686e03c922417fbf02de7292f97721"}
2022/04/20 13:53:15 [DEBUG] Attempting to open CLI config file: /Users/zobairq/.terraformrc
2022/04/20 13:53:15 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2022/04/20 13:53:15 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2022/04/20 13:53:15 [DEBUG] ignoring non-existing provider search directory /Users/zobairq/.terraform.d/plugins
2022/04/20 13:53:15 [DEBUG] ignoring non-existing provider search directory /Users/zobairq/Library/Application Support/io.terraform/plugins
2022/04/20 13:53:15 [DEBUG] ignoring non-existing provider search directory /Library/Application Support/io.terraform/plugins
2022/04/20 13:53:15 [INFO] CLI command args: []string{"import", "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721", "01cc14d1b817a0abc6f8f278499b0633/4f686e03c922417fbf02de7292f97721"}
2022/04/20 13:53:15 [WARN] Log levels other than TRACE are currently unreliable, and are supported only for backward compatibility.
  Use TF_LOG=TRACE to see Terraform's internal logs.
  ----
2022/04/20 13:53:15 [DEBUG] checking for provisioner in "."
2022/04/20 13:53:15 [DEBUG] checking for provisioner in "/usr/local/bin"
2022/04/20 13:53:16 [INFO] Failed to read plugin lock file .terraform/plugins/darwin_amd64/lock.json: open .terraform/plugins/darwin_amd64/lock.json: no such file or directory
2022/04/20 13:53:16 [DEBUG] POST https://gitlab.cego.dk/api/v4/projects/1282/terraform/state/spilnu-cloudflare/lock
2022/04/20 13:53:16 [DEBUG] GET https://gitlab.cego.dk/api/v4/projects/1282/terraform/state/spilnu-cloudflare
2022/04/20 13:53:16 [DEBUG] backend/local: skipping refresh of managed resources
2022-04-20T13:53:16.324+0200 [INFO]  plugin: configuring client automatic mTLS
2022-04-20T13:53:16.348+0200 [DEBUG] plugin: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0]
2022-04-20T13:53:16.351+0200 [DEBUG] plugin: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 pid=59773
2022-04-20T13:53:16.352+0200 [DEBUG] plugin: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0
2022-04-20T13:53:16.361+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: configuring server automatic mTLS: timestamp=2022-04-20T13:53:16.361+0200
2022-04-20T13:53:16.378+0200 [DEBUG] plugin: using plugin: version=5
2022-04-20T13:53:16.378+0200 [DEBUG] plugin.terraform-provider-cloudflare_v3.13.0: plugin address: address=/var/folders/h1/j9b3446n4j319x72bv041jw40000gn/T/plugin3013309984 network=unix timestamp=2022-04-20T13:53:16.378+0200
2022-04-20T13:53:16.426+0200 [WARN]  plugin.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2022-04-20T13:53:16.427+0200 [DEBUG] plugin: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 pid=59773
2022-04-20T13:53:16.427+0200 [DEBUG] plugin: plugin exited
2022/04/20 13:53:16 [DEBUG] adding implicit provider configuration provider["registry.terraform.io/cloudflare/cloudflare"], implied first by cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721
2022/04/20 13:53:16 [DEBUG] ProviderTransformer: "cloudflare_notification_policy.terraform_managed_resource_58c27beae56f40b5b37a61c651337d04" (*terraform.NodeAbstractResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022/04/20 13:53:16 [DEBUG] ProviderTransformer: "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721" (*terraform.NodeAbstractResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022/04/20 13:53:16 [DEBUG] ProviderTransformer: "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721 (import id \"01cc14d1b817a0abc6f8f278499b0633/4f686e03c922417fbf02de7292f97721\")" (*terraform.graphNodeImportState) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022/04/20 13:53:16 [DEBUG] ProviderTransformer: "cloudflare_notification_policy.terraform_managed_resource_7f244059d2ca4234b1a65f3a7fe2e2df" (*terraform.NodeAbstractResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022/04/20 13:53:16 [DEBUG] ProviderTransformer: "cloudflare_notification_policy.terraform_managed_resource_ac27f9d957da4c6d929c40b62ea4dc36" (*terraform.NodeAbstractResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721 (import id \"01cc14d1b817a0abc6f8f278499b0633/4f686e03c922417fbf02de7292f97721\")" references: []
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" references: []
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "cloudflare_notification_policy.terraform_managed_resource_7f244059d2ca4234b1a65f3a7fe2e2df" references: []
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "cloudflare_notification_policy.terraform_managed_resource_ac27f9d957da4c6d929c40b62ea4dc36" references: []
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "cloudflare_notification_policy.terraform_managed_resource_58c27beae56f40b5b37a61c651337d04" references: []
2022/04/20 13:53:16 [DEBUG] ReferenceTransformer: "cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721" references: []
3a7fe2e2df - *terraform.NodeAbstractResource
    cloudflare_notification_policy.terraform_managed_resource_ac27f9d957da4c6d929c40b62ea4dc36 - *terraform.NodeAbstractResource
  root - *terraform.nodeCloseModule
    provider["registry.terraform.io/cloudflare/cloudflare"] (close) - *terraform.graphNodeCloseProvider
  ------
2022/04/20 13:53:16 [DEBUG] Starting graph walk: walkImport
2022-04-20T13:53:16.429+0200 [INFO]  plugin: configuring client automatic mTLS
2022-04-20T13:53:16.453+0200 [DEBUG] plugin: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0]
2022-04-20T13:53:16.457+0200 [DEBUG] plugin: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 pid=59774
2022-04-20T13:53:16.457+0200 [DEBUG] plugin: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0
2022-04-20T13:53:16.464+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: configuring server automatic mTLS: timestamp=2022-04-20T13:53:16.464+0200
2022-04-20T13:53:16.482+0200 [DEBUG] plugin.terraform-provider-cloudflare_v3.13.0: plugin address: address=/var/folders/h1/j9b3446n4j319x72bv041jw40000gn/T/plugin3322551650 network=unix timestamp=2022-04-20T13:53:16.482+0200
2022-04-20T13:53:16.482+0200 [DEBUG] plugin: using plugin: version=5
2022-04-20T13:53:16.528+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:16 [INFO] Cloudflare Client configured for user: zoq@cego.dk: timestamp=2022-04-20T13:53:16.528+0200
cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721: Importing from ID "01cc14d1b817a0abc6f8f278499b0633/4f686e03c922417fbf02de7292f97721"...
2022-04-20T13:53:16.530+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:16 [DEBUG] Cloudflare API Request Details:
---[ REQUEST ]---------------------------------------
GET /client/v4/accounts/01cc14d1b817a0abc6f8f278499b0633/alerting/v3/policies/4f686e03c922417fbf02de7292f97721 HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/0.14.7 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/3.13.0
Content-Type: application/json
Accept-Encoding: gzip


-----------------------------------------------------: timestamp=2022-04-20T13:53:16.530+0200
2022-04-20T13:53:17.078+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:17 [DEBUG] Cloudflare API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Cf-Cache-Status: DYNAMIC
Cf-Ray: 6fedadb73c0010c5-CPH
Content-Type: application/json; charset=UTF-8
Date: Wed, 20 Apr 2022 11:53:17 GMT
Expect-Ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGU56Rb8iNWZVUvXhob6SvRADV1f; SameSite=Lax; path=/; expires=Wed, 20-Apr-22 14:23:18 GMT; HttpOnly
Set-Cookie: __cfruid=9918c5681699b98e4392c2826c28f9665d823142-1650455597; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Vary: Accept-Encoding
X-Envoy-Upstream-Service-Time: 382

{
  "result": {
    "id": "4f686e03c922417fbf02de7292f97721",
    "name": "Origin Error Rate Alert",
    "description": "",
    "enabled": true,
    "alert_type": "http_alert_origin_error",
    "mechanisms": {
      "pagerduty": [
        {
          "name": "cegoas: Spilnu Cloudflare",
          "id": "080395b4-74db-4d69-8c99-352159f7bf4a"
        }
      ]
    },
    "created": "2021-11-18T08:48:06.759786Z",
    "modified": "2021-11-18T08:48:06.759786Z",
    "conditions": {
      "and": [
        {
          "or": [
            {
              "==": [
                {
                  "var": "zones"
                },
"bd629a061d48aa41a84e74334d47aab3"
              ]
            },
            {
              "==": [
                {
                  "var": "zones"
                },
"738d29ca63cfb7570ab335ce428adaf5"
              ]
            },
            {
              "==": [
                {
                  "var": "zones"
                },
"68721a491e957bfd84cd76fdda33c651"
              ]
            }
          ]
        },
        {
          "or": [
            {
              "==": [
                {
                  "var": "slo"
                },
"99.9"
              ]
            }
     ]
        }
      ]
    },
    "filters": {
      "slo": [
"99.9"
      ],
      "zones": [
        "bd629a061d48aa41a84e74334d47aab3",
        "738d29ca63cfb7570ab335ce428adaf5",
"68721a491e957bfd84cd76fdda33c651"
      ]
    }
  },
  "success": true,
  "errors": [],
  "messages": []
}

-----------------------------------------------------: timestamp=2022-04-20T13:53:17.078+0200
2022-04-20T13:53:17.078+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:17 [ERROR] setting state: Invalid address to set: []string{"filters", "0", "slo"}: timestamp=2022-04-20T13:53:17.078+0200
cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721: Import prepared!
2022-04-20T13:53:17.080+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:17 [DEBUG] Cloudflare API Request Details:
---[ REQUEST ]---------------------------------------
GET /client/v4/accounts/01cc14d1b817a0abc6f8f278499b0633/alerting/v3/policies/4f686e03c922417fbf02de7292f97721 HTTP/1.1
Host: api.cloudflare.com
User-Agent: terraform/0.14.7 terraform-plugin-sdk/2.10.1 terraform-provider-cloudflare/3.13.0
Content-Type: application/json
Accept-Encoding: gzip


-----------------------------------------------------: timestamp=2022-04-20T13:53:17.080+0200
  Prepared cloudflare_notification_policy for import
cloudflare_notification_policy.terraform_managed_resource_4f686e03c922417fbf02de7292f97721: Refreshing state... [id=4f686e03c922417fbf02de7292f97721]
2022-04-20T13:53:17.629+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:17 [DEBUG] Cloudflare API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/2.0 200 OK
Cf-Cache-Status: DYNAMIC
Cf-Ray: 6fedadbaa800d88d-CPH
Content-Type: application/json; charset=UTF-8
Date: Wed, 20 Apr 2022 11:53:17 GMT
Expect-Ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
Server: cloudflare
Set-Cookie: __cflb=0H28vgHxwvgAQtjUGU56Rb8iNWZVUvXhoko7wDyi6HB; SameSite=Lax; path=/; expires=Wed, 20-Apr-22 14:23:18 GMT; HttpOnly
Set-Cookie: __cfruid=9918c5681699b98e4392c2826c28f9665d823142-1650455597; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
Vary: Accept-Encoding
X-Envoy-Upstream-Service-Time: 371

{
  "result": {
    "id": "4f686e03c922417fbf02de7292f97721",
    "name": "Origin Error Rate Alert",
    "description": "",
    "enabled": true,
    "alert_type": "http_alert_origin_error",
    "mechanisms": {
      "pagerduty": [
        {
          "name": "cegoas: Spilnu Cloudflare",
          "id": "080395b4-74db-4d69-8c99-352159f7bf4a"
        }
      ]
    },
    "created": "2021-11-18T08:48:06.759786Z",
    "modified": "2021-11-18T08:48:06.759786Z",
    "conditions": {
      "and": [
        {
          "or": [
            {
              "==": [
                {
                  "var": "zones"
                },
"bd629a061d48aa41a84e74334d47aab3"
              ]
            },
            {
              "==": [
                {
                  "var": "zones"
                },
"738d29ca63cfb7570ab335ce428adaf5"
              ]
            },
            {
              "==": [
                {
                  "var": "zones"
                },
"68721a491e957bfd84cd76fdda33c651"
              ]
            }
          ]
        },
        {
          "or": [
            {
              "==": [
                {
                  "var": "slo"
                },
"99.9"
              ]
            }
     ]
        }
      ]
    },
    "filters": {
      "slo": [
"99.9"
      ],
      "zones": [
        "bd629a061d48aa41a84e74334d47aab3",
        "738d29ca63cfb7570ab335ce428adaf5",
"68721a491e957bfd84cd76fdda33c651"
      ]
    }
  },
  "success": true,
  "errors": [],
  "messages": []
}

-----------------------------------------------------: timestamp=2022-04-20T13:53:17.629+0200
2022-04-20T13:53:17.629+0200 [INFO]  plugin.terraform-provider-cloudflare_v3.13.0: 2022/04/20 13:53:17 [ERROR] setting state: Invalid address to set: []string{"filters", "0", "slo"}: timestamp=2022-04-20T13:53:17.629+0200

2022/04/20 13:53:17 [DEBUG] DELETE https://gitlab.cego.dk/api/v4/projects/1282/terraform/state/spilnu-cloudflare/lock
Error: failed to set filters: Invalid address to set: []string{"filters", "0", "slo"}


2022-04-20T13:53:17.723+0200 [WARN]  plugin.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2022-04-20T13:53:17.725+0200 [DEBUG] plugin: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/3.13.0/darwin_amd64/terraform-provider-cloudflare_v3.13.0 pid=59774
2022-04-20T13:53:17.725+0200 [DEBUG] plugin: plugin exited
```
### Panic output

Error: failed to set filters: Invalid address to set: []string{"filters", "0", "slo"}

### Expected output

Import successful!

### Actual output

Error: failed to set filters: Invalid address to set: []string{"filters", "0", "slo"}

### Steps to reproduce

1. Create a notifaction policy with filters slo, and 3 zones
2. create the resource 
3. import it 
<img width="959" alt="billede" src="https://user-images.githubusercontent.com/14279865/164223757-388f357b-2b2c-418b-9c95-e94723ad9a13.png">


### Additional factoids

_No response_

### References

_No response_

--- a/.changelog/1573.txt
+++ b/.changelog/1573.txt
@@ -1,0 +1,3 @@
+```release-note:enhancement
+resource/cloudflare_notification_policy: Add `slo` to notification policy filters
+```

--- a/cloudflare/schema_cloudflare_notification_policy.go
+++ b/cloudflare/schema_cloudflare_notification_policy.go
@@ -125,6 +125,13 @@ func notificationPolicyFilterSchema() *schema.Schema {
 					Optional:     true,
 					RequiredWith: []string{"filters.0.enabled"},
 				},
+				"slo": {
+					Type: schema.TypeSet,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
+					},
+					Optional: true,
+				},
 			},
 		},
 	}

--- a/website/docs/r/notification_policy.html.markdown
+++ b/website/docs/r/notification_policy.html.markdown
@@ -89,6 +89,7 @@ The following arguments are supported:
 * `real_origin_monitoring` - (Optional) Cloudflare is unable to reach your origin (refer to the [nested schema](#nestedblock--alert-type-real-origin-monitoring)).
 * `universal_ssl_event_type` - (Optional) Universal certificate notices (refer to the [nested schema](#nestedblock--alert-type-universal-ssl-event-type)).
 * `bgp_hijack_notification` - (Optional) Alerts for BGP hijack (refer to the [nested schema](#nestedblock--alert-type-bgp-hijack-notification)).
+* `http_alert_origin_error` - (Optional) HTTP origin error rate alert  (refer to the [nested schema](#nestedblock--alert-type-http-alert-origin-error)).
 
 <a id="nestedblock--alert-type-billing-usage-alert"></a>
 **Nested schema for `billing_usage_alert`**
@@ -116,6 +117,13 @@ The following arguments are supported:
 
 <a id="#nestedblock--alert-type-bgp-hijack-notification"></a>
 **Nested schema for `bgp_hijack_notification`**
+
+<a id="nestedblock--alert-type-http-alert-origin-error"></a>
+**Nested schema for `http_alert_origin_error`**
+
+* `zones` - (Optional) A list of zone identifiers.
+* `slo` - (Optional) A numerical limit. Example: `"99.9"`
+
 
 ## Import
 

